unicorn/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+INCLUDE_DIRECTORIES(
+    ${PROJECT_SOURCE_DIR}/unicorn/include)
+
+ADD_LIBRARY(unicorn MODULE
+    include/cocaine/unicorn.hpp
```

Comment:
Header files aren't required to be listed here to be able to successful build the plugin.